platform/ios/MGLMapView.mm

```diff
@@ -82,6 +82,7 @@ @interface MGLMapView () <UIGestureRecognizerDelegate, GLKViewDelegate, CLLocati
 @property (nonatomic, getter=isDormant) BOOL dormant;
 @property (nonatomic, getter=isAnimatingGesture) BOOL animatingGesture;
 @property (nonatomic, readonly, getter=isRotationAllowed) BOOL rotationAllowed;
+@property (nonatomic, readonly, getter=hasSettingsBundleMetricsOptOut) BOOL settingsBundleMetricsOptOut;
```

Comment:
The property itself should be called `hasSettingsBundleMetricsOptOut`, because it doesn’t hold the opt-out control but only indicates its existence.